dali/server/daserver.cpp

```diff
@@ -587,8 +590,9 @@ int main(int argc, char* argv[])
         Owned<IMPServer> mpServer = getMPServer();
         Owned<IWhiteListHandler> whiteListHandler = createWhiteListHandler(populateWhiteListFromEnvironment, formatDaliRole);
         mpServer->installWhiteListCallback(whiteListHandler);
-
+#ifndef _CONTAINERIZED
         setMsgLevel(fileMsgHandler, serverConfig->getPropInt("SDS/@msgLevel", 100));
```

Comment:
in container world, can make this property legacy, and if using standard routine, pick up a common property e.g. logging/loggingLevel